lock/lock.py

```diff
@@ -287,56 +279,52 @@ async def unlock(
         `[p]unlock #general`
         `[p]unlock 739562845027353 true`
         """
-        try:
-            await ctx.trigger_typing()
```

Comment:
Please use `await ctx.typing()` and retain the try/except block rather than using a context manager.